gpt4all-bindings/golang/binding.cpp

```diff
@@ -19,46 +16,24 @@
 #include <iostream>
 #include <unistd.h>
 
-void* load_mpt_model(const char *fname, int n_threads) {
+void* load_gpt4all_model(const char *fname, int n_threads) {
```

Comment:
Why *`gpt4all`*`_model`?

Reply:
I think just calling it `load_model` would be better, since having `gpt4all` in the name makes it sound like it's for gpt4all models only.